### PR TITLE
Auto sync

### PR DESCRIPTION
**Pull request by robinmonjo**
Initial auto sync version. It:
- works on both local and production environment
- it auto sync the changes on Eventmaker

Next step:
- publish this version
- integrate it into eventmaker-cms-theme
- have a new "beta" expose and tell Eventmaker developers to give it a try

### CODE REVIEW

File: bin/expose.js [hunk 1]
```diff
@@ -1,3 +1,168 @@
...
+      demandOption: true
+    },
+    "eventId": {
+      alias: "e",
+      describe: "The Eventmaker event id you are working on",
+      demandOption: true
+    },
+    "initialSync": {
+      alias: "s",
+      describe: "Wether or not the theme will be entirely synced on launch",
```
Comment (opheJansen) [inline bin/expose.js:38]: You mean `Whether` not `wether` ([mouton](https://www.wordreference.com/enfr/wether)) ?

File: bin/expose.js [hunk 2]
```diff
@@ -1,3 +1,168 @@
...
+      boolean: true,
+      default: false
+    },
+    "port": {
+      alias: "p",
+      describe: "the port used by the local server",
```
Comment (opheJansen) [inline bin/expose.js:44]: It nothing much but keep the majuscule to start all description ^^

File: bin/expose.js [hunk 3]
```diff
@@ -1,3 +1,168 @@
...
+    "watchPath": {
+      describe: "Auto sync folder",
+      default: "./themes/**/*.liquid"
+    },
+    "devServer": {
+      describe: "The Webpack Dev Server endpoint",
+      default: "http://localhost:9999"
+    },
+    "eventmakerLocalEndpoint": {
+      describe: "when using local=true specify the local endpoint of Eventmaker",
```
Comment (opheJansen) [inline bin/expose.js:62]: same for the majuscule

File: bin/expose.js [hunk 4]
```diff
@@ -1,3 +1,168 @@
...
+  });
+}
+
+const summary = (theme, host) => {
+  log("✅ everything is ready 😊");
+  log(`  - working on ${theme} on event ${eventId}`);
+  log(`  - local server running at ${host}`);
+  log(`  - webpack server running at ${devServer}`);
+  initialSync ? (
+    log(`  - local theme has been reloaded, assets will be fetched from the local server}`)
```
Comment (opheJansen) [inline bin/expose.js:146]: why the `}` at the end ?

File: bin/expose.js [hunk 5]
```diff
@@ -1,3 +1,168 @@
...
+  logUpload,
+  logInfo,
+  wrapRed,
+  logHelp
+} = require("../lib/utils/log");
+
+const argv = yargs(hideBin(process.argv))
+  .options({
+    "token": {
+      alias: "a",
```
Comment (laurent-d) [inline bin/expose.js:27]: Perhaps t instead of a

**Pull request merged**